middleware/src/rpcmessages/rpcmessages.go

```diff
@@ -53,3 +88,21 @@ type VerificationProgressResponse struct {
 	Headers              int64   `json:"headers"`
 	VerificationProgress float64 `json:"verificationProgress"`
 }
+
+// FlashdriveResponse is the struct that gets sent by the rpc server during a Flashdrive call
+type FlashdriveResponse struct {
+	Success bool
+	Message string
+}
+
+// BackupResponse is the struct that gets sent by the rpc server during a Backup call
+type BackupResponse struct {
+	Success bool
+	Message string
+}
+
+// BackupResponse is the struct that gets sent by the rpc server during a Backup call
+type RestoreResponse struct {
+	Success bool
+	Message string
+}
```

Comment:
Do you think it's necessary to have three different response types for this that are the same?
The whole procedure of plugging in a flashdrive and backing up or restoring is to me part of the same concept so I think could be collapsed into one response type. You already have a message field so additional details could just go there?
 Just a minor nit :)

Reply:
> Do you think it's necessary to have three different response types for this that are the same?

I thought about this as well beforehand, and I don't think it's necessary. Might replace it by a `GenericResponse` which could be used by even more RPCs, than the Flashdrive, Backup and Restore RPCs, that might get added in the future. Good point!